plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -307,6 +320,21 @@ function OPDSBrowser:parseFeed(item_url)
     end
 end
 
+function OPDSBrowser:getServerFileName(item_url)
+    local headers = self:fetchFeed(item_url, true)
+    if headers then
+        logger.dbg("OPDSBrowser: server file headers", headers)
```

Comment:
I was thinking we should add something like this, maybe in socketutil although it's not really about sockets.

```lua
local function log_safe_headers(headers)
    local sensitive_headers = {
        ["authorization"] = true,
        ["cookie"] = true,
        ["set-cookie"] = true,
        ["proxy-authorization"] = true
    }
    local safe_headers = {}
    for key, value in pairs(headers) do
        if sensitive_headers[key] then
            safe_headers[key] = "REDACTED"
        else
            safe_headers[key] = value
        end
    end
    logger.dbg("OPDSBrowser: server file headers", safe_headers)
end
```